R/derive_var_extreme_flag.R

```diff
@@ -231,6 +231,12 @@ derive_var_extreme_flag <- function(dataset,
 }
 
 #' Adds a Variable Flagging the Maximal / Minimal Value Within a Group of Observations
+#' @description
+#' `r lifecycle::badge("deprecated")`
+#'
+#' This function is *deprecated*. Please use `restrict_derivation()` / `derive_var_extreme_flag()`
```

Comment:
Should be `slice_derivation()` instead of `restrict_derivation()`. And I think the `order` argument should be adjusted instead of the `mode` argument.